circle.yml

```diff
@@ -15,18 +15,12 @@ notify:
     - url: https://webhooks.stackstorm.net:8531/webhooks/build/events
 
 machine:
-  # Overwrite these ENV variables in parametrized (manual/API) builds
   environment:
     DISTROS: "wheezy jessie trusty xenial el6 el7"
     NOTESTS: "xenial el7"
     ST2_GITURL: https://github.com/StackStorm/st2
-    ST2_GITREV: master
```

Comment:
I do get general tone of your concern, I also do like the readability this section provides. In this particular case, though, we're talking git branch name, there should be very little confusion as to what default is.